src/components/document/document.tsx

```diff
@@ -367,11 +367,13 @@ export class DocumentComponent extends BaseComponent<IProps, IState> {
   }
 
   private handleComment = () => {
-    const { ui: { selectedTileId } } = this.stores;
-    if (selectedTileId ) {
+    // TODO: how to handle comments when multiple tiles are selected?
+    //       currently it will select the first tile for commenting
```

Comment:
This makes sense as the implementation here, but as I mentioned in the PT story my inclination would be to disable the UI for commenting (and any other inherently single-tile operation) when there are multiple tiles selected. This could easily lead to the wrong tile being commented from the user's perspective, as this will comment the first selected tile whereas users might expect it to be the most recently selected tile.